### PR TITLE
Update default helm image to latest

### PR DESCRIPTION
**Pull request by xiangfu0**
Update default helm docker image to `latest` since the `latest-jdk11` is not published anymore.

### CODE REVIEW

File: kubernetes/helm/pinot/values.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@
 
 image:
   repository: apachepinot/pinot
-  tag: latest-jdk11 # release-0.7.1
-  pullPolicy: IfNotPresent
+  tag: latest # release-0.7.1
```
Comment (Jackie-Jiang) [inline kubernetes/helm/pinot/values.yaml:24]: Does `0.7.1` still apply?

File: kubernetes/helm/pinot/values.yaml [hunk 2]
```diff
@@ -21,8 +21,8 @@
...
+  pullPolicy: Always
```
Comment (Jackie-Jiang) [inline kubernetes/helm/pinot/values.yaml:25]: Why do we need to change this

**Review by Jackie-Jiang — APPROVED** [review]

File: kubernetes/helm/pinot/values.yaml (hunk 1 above)
Comment (xiangfu0) [reply kubernetes/helm/pinot/values.yaml:24]: It's just an example, let me also make it to be more update.

File: kubernetes/helm/pinot/values.yaml (hunk 2 above)
Comment (xiangfu0) [reply kubernetes/helm/pinot/values.yaml:25]: This is to let the latest image always up to date

**Pull request merged**